torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -216,10 +216,10 @@ def _build_tensor(size, value=None, dtype=torch.float):
     return torch.empty(size, size, size, dtype=dtype).fill_(value)
 
 
-def _build_multidim_tensor(dim, dim_size, value=None):
+def _build_multidim_tensor(dim, dim_size, value=None, dtype=torch.float):
     if value is None:
         value = size
-    return torch.FloatTensor(size=[dim_size for _ in range(dim)]).fill_(value)
```

Comment:
What's the use case for typed Tensor classes that are typed on their dtype, vs. the generic torch.Tensor class?

Reply:
not sure about the history here, but I think your solution is better.